src/zc/buildout/easy_install.py

```diff
@@ -1152,6 +1152,8 @@ def _relative_path(common, path):
 def _relativitize(path, script, relative_paths):
     if path == script:
         raise AssertionError("path == script")
+    if path == relative_paths:
+        return "base"
```

Comment:
Ah, you're right. Below there's `"join(base, %r)"`, so `"base"` is indeed the correct syntax.